test/xspec-tvt.xspec

```diff
@@ -0,0 +1,296 @@
+			test="$x:result/href-doc-child/@expand-text/string()" />
+	</x:scenario>
+
+	<!--
+		Scenarios applicable to XSLT and XQuery (but not implemented yet for XQuery)
+	-->
+
+	<x:scenario label="function-param">
+		<x:scenario label="user-content">
+			<x:call function="one-or-more">
+				<x:param>
+					<function-param-child x:expand-text="yes">}}{false()}{{</function-param-child>
+					<function-param-child expand-text="yes">}}{false()}{{</function-param-child>
+				</x:param>
+			</x:call>
+			<x:like label="user-content expect" />
+		</x:scenario>
+
+		<x:scenario label="@href">
+			<x:call function="exactly-one">
+				<x:param href="xspec-tvt.xml" />
+			</x:call>
+			<x:like label="href expect" />
+		</x:scenario>
+	</x:scenario>
+
+	<x:param name="global-param-user-content">
+		<global-param-child x:expand-text="yes">}}{false()}{{</global-param-child>
+		<global-param-child expand-text="yes">}}{false()}{{</global-param-child>
+	</x:param>
+	<x:param href="xspec-tvt.xml" name="global-param-href" />
+	<x:scenario label="global-param">
+		<x:scenario label="user-content">
+			<x:call function="one-or-more">
+				<x:param select="$global-param-user-content" />
+			</x:call>
+			<x:like label="user-content expect" />
+		</x:scenario>
+
+		<x:scenario label="@href">
+			<x:call function="exactly-one">
+				<x:param select="$global-param-href" />
+			</x:call>
+			<x:like label="href expect" />
+		</x:scenario>
+	</x:scenario>
+
+	<x:variable name="global-variable-user-content">
+		<global-variable-child x:expand-text="yes">}}{false()}{{</global-variable-child>
+		<global-variable-child expand-text="yes">}}{false()}{{</global-variable-child>
+	</x:variable>
+	<x:variable href="xspec-tvt.xml" name="global-variable-href" />
+	<x:scenario label="global variable">
+		<x:scenario label="user-content">
+			<x:call function="one-or-more">
+				<x:param select="$global-variable-user-content" />
+			</x:call>
+			<x:like label="user-content expect" />
+		</x:scenario>
+
+		<x:scenario label="@href">
+			<x:call function="exactly-one">
+				<x:param select="$global-variable-href" />
+			</x:call>
+			<x:like label="href expect" />
+		</x:scenario>
+	</x:scenario>
+
+	<x:scenario label="local variable">
+		<x:variable name="local-variable-user-content">
+			<local-variable-child x:expand-text="yes">}}{false()}{{</local-variable-child>
+			<local-variable-child expand-text="yes">}}{false()}{{</local-variable-child>
+		</x:variable>
+		<x:variable href="xspec-tvt.xml" name="local-variable-href" />
+		<x:scenario label="user-content">
+			<x:call function="one-or-more">
+				<x:param select="$local-variable-user-content" />
+			</x:call>
+			<x:like label="user-content expect" />
+		</x:scenario>
+
+		<x:scenario label="@href">
+			<x:call function="exactly-one">
+				<x:param select="$local-variable-href" />
+			</x:call>
+			<x:like label="href expect" />
+		</x:scenario>
+	</x:scenario>
+
+	<x:scenario label="assertion">
+		<x:scenario label="user-content">
```

Comment:
The following assertion also works. Is it worth adding to this scenario?

```
<x:expect label="@expand-text=yes on x:expect enables TVT for its child text node"
select="text()/string()" test="'}false{'" expand-text="yes">}}{false()}{{</x:expect>
```

Reply:
Added tests in 235d685e373ce4d5f14d3943831e064f1fc326d0 (XSLT) and 9e81a61410761e85720958b77ca4354ad135b81d (Schematron).